include/drivers/flash.h

```diff
@@ -296,6 +299,28 @@ static inline size_t z_impl_flash_get_write_block_size(struct device *dev)
 	return api->write_block_size;
 }
 
+
+/**
+ *  @brief  Get value that represents erased flash cell
+ *
+ *  @param dev flash device
+ *
+ *  @return value that represents erased flash cell
+ */
+__syscall u8_t flash_get_erase_value(const struct device *dev);
+
+static inline u8_t z_impl_flash_get_erase_value(const struct device *dev)
+{
+#if defined(CONFIG_FLASH_ERASE_VAL)
+	const struct flash_driver_api *api =
+		(const struct flash_driver_api *)dev->driver_api;
+
+	return api->erase_value;
+#else
```

Comment:
else not needed - introduce this API to all driver instead.

Reply:
So what you mean is: this is not configurable all drivers fill in the erase_value as an requirement?

Reply:
^^ yes

Reply:
similar as FLASH_PAGE_LAYOUT

Reply:
OK, if for example the NVS is in use, which requires to know the erase value, then this should be enabled by the dependency, etc?